converter/src/main/java/gov/cms/qpp/conversion/util/DuplicationCheckHelper.java

```diff
@@ -0,0 +1,28 @@
+package gov.cms.qpp.conversion.util;
+
+import gov.cms.qpp.conversion.decode.AggregateCountDecoder;
+import gov.cms.qpp.conversion.model.Node;
+
+import java.util.List;
+
+/**
+ * Utility class to return the number of duplications found.
+ */
+public class DuplicationCheckHelper {
+	public static final int ACCOUNT_FOR_ORIGINAL_AGGREGATE_COUNT = 1;
+	public static final int ACCOUNT_FOR_MISSING_AGGREGATE_COUNT = 0;
+
+	private DuplicationCheckHelper() {}
+
+	/**
+	 * Finds the number of duplications of aggregate counts
+	 *
+	 * @param node to check duplications from
+	 * @return
+	 */
+	public static int calculateDuplications(Node node) {
```

Comment:
I would make this method name more specific to mention that it is looking for duplicate `aggregateCount`s.

Reply:
Or make this method take in a `TemplateId` and then you could use this helper method in `Checker` too?